### PR TITLE
Update wallet docs

### PR DESCRIPTION
**Pull request by nop33**
@mvaivre I started improving the docs we have about the wallet, I would like to, later on, do another pass once we are ready for the v2.0 release. This is just a draft for now.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -276,14 +276,15 @@ Theoretically yes, the future is not really projectable beyond 80 years. However
...
+### What analytics does the desktop wallet collect?
+
+In Alephium, we take your concerns about privacy and user experience seriously. Enabling analytics can actually help us improve your experience without compromising your privacy. Rest assured that the information we collect is completely anonymous. When you launch your wallet for the first time, we generate a unique ID (for example, `vCJGCsDPrZ8WJaIKZMWjU`) which is the only identification information we require. We don't collect IPs or any other [personal data](https://posthog.com/blog/what-is-personal-data-pii). We only collect data on button clicks, number of wallets, number of addresses, number of contacts, and wallet preferences. This helps us understand which features are useful to our users and which ones are not. We encourage you to check out our open-source code base and [look for the `posthog?.capture` keyword](https://github.com/alephium/desktop-wallet/search?q=posthog%3F.capture) to see exactly which events we capture.
```
Comment (MaudSimon) [inline docs/FAQ.md]: Could we rephrase without using "we"? Like this for example: 

Alephium takes concerns about privacy and user experience seriously. Enabling analytics can actually help improve your experience without compromising your privacy. The information collected by the Desktop wallet is completely anonymous. When you launch your wallet for the first time, a unique ID is generated (for example, `vCJGCsDPrZ8WJaIKZMWjU`) which is the only identification information required. IPs or any other [personal data](https://posthog.com/blog/what-is-personal-data-pii) are not collected. The only data collected are button clicks, number of wallets, number of addresses, number of contacts, and wallet preferences. It helps understand which features are useful to users and which ones are not. Alephium encourages everyone to check out its open-source code base and [look for the `posthog?.capture` keyword](https://github.com/alephium/desktop-wallet/search?q=posthog%3F.capture) to see exactly which events are captured.

File: docs/FAQ.md [hunk 2]
```diff
@@ -276,14 +276,15 @@ Theoretically yes, the future is not really projectable beyond 80 years. However
 
 ### What type of wallet do you offer? Is it planned to offer hardware wallets?
 
-We currently have a desktop [wallet](https://github.com/alephium/desktop-wallet/releases/latest) and the team is working on mobile and [web extension wallets](https://github.com/alephium/extension-wallet/releases/latest). We want to offer the possibility to store ALPH on hardware wallets. We are currently pursuing being supported on Ledger. Please note that it is a lengthy process and won't be completed overnight.
+We currently have a [desktop wallet](https://github.com/alephium/desktop-wallet/releases/latest) and a [web extension wallet](https://github.com/alephium/extension-wallet/releases/latest). The team is actively working on a [mobile wallet](https://github.com/alephium/mobile-wallet). We want to offer the possibility to store ALPH on hardware wallets. We are currently pursuing being supported on Ledger. Please note that it is a lengthy process and won't be completed overnight.
```
Comment (MaudSimon) [inline docs/FAQ.md]: Alephium currently offers: 
- a [desktop wallet](https://github.com/alephium/desktop-wallet/releases/latest) 
- a [web extension wallet](https://github.com/alephium/extension-wallet/releases/latest) available on [Chrome] (https://chrome.google.com/webstore/detail/alephium-extension-wallet/gdokollfhmnbfckbobkdbakhilldkhcj) and [Firefox] (https://addons.mozilla.org/en-US/firefox/addon/alephiumextensionwallet/)

A [mobile wallet](https://github.com/alephium/mobile-wallet) is currently being worked on. 

Having Alephium available on hardware wallets is an important priority for the project. Ledger support is currently being worked on. Please note, it is a lengthy process with multiple stages that will take time to complete.

Comment (nop33) [reply docs/FAQ.md]: Updated

File: docs/FAQ.md (hunk 1 above)
Comment (nop33) [reply docs/FAQ.md]: Updated

**Pull request merged**